### PR TITLE
Update to ODK docs

### PR DESCRIPTION
**Pull request by matentzn**

### CODE REVIEW

File: docs/editors.md [hunk 1]
```diff
@@ -1,2 +1,31 @@
...
+
+## Run a COB release:
+
+See standard ODK release docs [here](odk-workflows/ReleaseWorkflow.md).
+
+## Run COB integration tests:
+
+### All COB tests:
+```
+cd src/ontology
```
Comment (cthoyt) [inline docs/editors.md:15]: why isn't this available as a top-level command from the makefile?

Comment (matentzn) [reply docs/editors.md:15]: There is no more Makefile up top, it was just an oversight.. it moved down in src/ontology.

Comment (cthoyt) [reply docs/editors.md:15]: IMO the lack of a top-level file like this makes it much less accessible. Why not just have a top-level make file that just does the things for the basic COB tests and regular ontology QC tests (and anything else)?

Comment (matentzn) [reply docs/editors.md:15]: I am totally open to that - the reason why its not is that 20-30 repos out there have that exact structure with the exact same commands and all, that we cant really move them out from there; and then the question is what is more confusing: having two makefiles, and therefore to different sets of documentation (remember, all the docs for src/ontology are fully autogenerated), or having one that is hidden - and only supposed to be used by the COB maintainers anyways.

Comment (cthoyt) [reply docs/editors.md:15]: hmm interesting thing to consider. It's not a blocker for this PR, though

**Review by cmungall — APPROVED** [review]

**Pull request merged**